Modules/_ssl.c

```diff
@@ -310,6 +310,10 @@ typedef struct {
     PyObject *set_hostname;
 #endif
     int check_hostname;
+    /* OpenSSL does not provide X509_VERIFY_PARAM_get_hostflags(), hostflags
+     * are initialized as 0.
```

Comment:
I don't understand this comment

Reply:
OpenSSL has no API to get hostflags from X509_VERIFY_PARAM. We have to maintain our own copy.

Reply:
Ahhh. Now I understand. I think "OpenSSL does not have an API for getting the hostflags from an X509_VERIFY_PARAM" would be clearer.

Also is there a bug/PR on OpenSSL for this we can link?